src/rgw/rgw_pubsub_push.cc

```diff
@@ -173,6 +173,40 @@ class RGWPubSubAMQPEndpoint : public RGWPubSubEndpoint {
   ack_level_t ack_level;
   amqp::connection_ptr_t conn;
 
+  bool get_verify_ssl(const RGWHTTPArgs& args) {
+    bool exists;
+    auto str_verify_ssl = args.get("verify-ssl", &exists);
+    if (!exists) {
+      // verify server certificate by default
+      return true;
+    }
+    boost::algorithm::to_lower(str_verify_ssl);
+    if (str_verify_ssl == "true") {
+      return true;
+    }
+    if (str_verify_ssl == "false") {
+      return false;
+    }
+    throw configuration_error("'verify-ssl' must be true/false, not: " + str_verify_ssl);
+  }
+
+  bool get_use_ssl(const RGWHTTPArgs& args) {
```

Comment:
see: https://github.com/ceph/ceph/pull/39392/files#r573597276
I think that we don't need this function